src/Raven.Server/Documents/Indexes/Static/IndexCompiler.cs

```diff
@@ -54,6 +54,8 @@ public static class IndexCompiler
         [ThreadStatic]
         private static bool DisableMatchingAdditionalAssembliesByNameValue;
 
+        private static long IndexVersion { get; set; }
```

Comment:
Why static? Don't we create each index at database startup? For example, it can lead to race if you start several indexes at the same time.
cc @arekpalinski @ppekrol

Reply:
Fixed